### PR TITLE
SFR-2200: Update license page to copyright

### PR DESCRIPTION
**Pull request by jackiequach**
[Jira Ticket](https://newyorkpubliclibrary.atlassian.net/browse/SFR-2200)

### This PR does the following:
- Updates license page to copyright and adds the In Copyright section

### Testing requirements & instructions: 
-

### CODE REVIEW

File: src/components/Copyright/Copyright.tsx [hunk 1]
```diff
@@ -159,6 +159,17 @@ const License: React.FC = () => {
...
+      <p>
+        Works that are{" "}
+        <Link to="https://rightsstatements.org/page/InC/1.0/?language=en">
+          In Copyright
+        </Link>{" "}
+        are protected by copyright and/or related rights. You are free to use
+        this Item in any way that is permitted by the copyright and related
+        rights legislation that applies to your use. For other uses you need to
+        obtain permission from the rights-holder(s).
+      </p>
```
Comment (kylevillegas93) [inline src/components/Copyright/Copyright.tsx:172]: I think this is ok to leave as is but it requires us to change this language every time we want to update the Copyright page. Is there a more robust test that we can write that is more agnostic to the content of the page?

Comment (kylevillegas93) [inline src/components/Copyright/Copyright.tsx:172]: I think this is ok to leave as is but it requires us to change this language every time we want to update the Copyright page. Is there a more robust test that we can write that is more agnostic to the content of the page?

File: src/components/EditionCard/LicenseLink.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Rights } from "~/src/types/DataModel";
+import Link from "~/src/components/Link/Link";
+
+const LicenseLink: React.FC<{ rights: Rights[] }> = ({ rights }) => {
+  return (
+    <Link to="/copyright">
+      {rights && rights.length > 0
+        ? `License: ${rights[0].rightsStatement}`
+        : "License: Unknown"}
```
Comment (kylevillegas93) [inline src/components/EditionCard/LicenseLink.tsx:10]: I'm seeing in the BRD that this should be Copyright but it's not reflected in the ticket. I recommend reaching out to Will to see if we want to include that in this change. I commented on the Jira ticket as well.

File: src/components/Copyright/Copyright.tsx (hunk 1 above)
Comment (kylevillegas93) [inline src/components/Copyright/Copyright.tsx:172]: Is there an updated BRD? I see that this is the language reflected in the ticket but not here: https://docs.google.com/document/d/12WVwIxC_OT3eWcw8-LQd1U-FX91tj-qWnL8A6d0CLsU/edit

Also https://rightsstatements.org/vocab/InC/1.0/ unfortunately is returning a Bad Gateway. I'd let Will know and have him make decision on whether we want to include it or not.

File: src/components/Copyright/__snapshots__/Copyright.test.tsx.snap [hunk 3]
```diff
@@ -316,6 +316,23 @@ exports[`renders License page unchanged 1`] = `
...
+        Works that are
+         
+        <a
+          class="chakra-link css-qsaw8"
+          href="https://rightsstatements.org/page/InC/1.0/?language=en"
+        >
+          In Copyright
+        </a>
+         
+        are protected by copyright and/or related rights. You are free to use this Item in any way that is permitted by the copyright and related rights legislation that applies to your use. For other uses you need to obtain permission from the rights-holder(s).
```
[the remark this answers is not in the document]
Comment (jackiequach) [reply src/components/Copyright/__snapshots__/Copyright.test.tsx.snap]: Added unit tests to test different parts of the page.

File: src/components/Copyright/Copyright.test.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { render } from "~/src/__tests__/testUtils/render";
+import { screen } from "@testing-library/react";
+import Copyright from "./Copyright";
+
+it("renders Copyright page unchanged", async () => {
```
Comment (kylevillegas93) [inline src/components/Copyright/Copyright.test.tsx:6]: Nice - I really like this test - ensures we have the right sections!

**Review by kylevillegas93 — APPROVED** [review]
Looks good!

**Pull request merged**